### PR TITLE
prefabs marketplace

### PR DESCRIPTION
**Pull request by swethakundaram**
1. Created a marketplace page for Prefabs. 
2. Added prefabs page to the top-navbar.
3. Adjusted basic styles.
4. Added all WM prefabs with new images and links to it.

To-do
1. Create another page and move all prefab contents on one page with a single sidebar.

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -11,13 +11,78 @@
...
+        caption: 'BoxOAuth',
+        // You will need to prepend the image path with your baseUrl
+        // if it is not '/', like: '/test-site/img/image.jpg'.
+        image: 'img/box.png',
+        infoLink: '/learn/app-development/widgets/prefab/box-viewer-prefab',
+        pinned: true,
+    },
+	
+		{
+        caption: 'GoogleOAuth',
```
Comment (prasanth-reddy-wavemaker-com) [inline website/siteConfig.js:50]: remove this from listing

File: website/siteConfig.js [hunk 2]
```diff
@@ -11,13 +11,78 @@
...
+        // You will need to prepend the image path with your baseUrl
+        // if it is not '/', like: '/test-site/img/image.jpg'.
+        image: 'img/google.png',
+        infoLink: '/learn/app-development/widgets/prefab/oauth-prefabs/google',
+        pinned: true,
+    },
+	
+		{
+        caption: 'Facebook',
```
Comment (prasanth-reddy-wavemaker-com) [inline website/siteConfig.js:59]: remove this from listing

File: website/siteConfig.js [hunk 3]
```diff
@@ -11,13 +11,78 @@
...
+        // You will need to prepend the image path with your baseUrl
+        // if it is not '/', like: '/test-site/img/image.jpg'.
+        image: 'img/facebook.png',
+        infoLink: '/learn/app-development/widgets/prefab/oauth-prefabs/facebook',
+        pinned: true,
+    },
+	
+		{
+        caption: 'Instagram',
```
Comment (prasanth-reddy-wavemaker-com) [inline website/siteConfig.js:68]: remove this from listing

File: website/siteConfig.js [hunk 4]
```diff
@@ -11,13 +11,78 @@
...
+        // You will need to prepend the image path with your baseUrl
+        // if it is not '/', like: '/test-site/img/image.jpg'.
+        image: 'img/instagram.png',
+        infoLink: '/learn/app-development/widgets/prefab/oauth-prefabs/instagram',
+        pinned: true,
+    },
+	
+		{
+        caption: 'LinkedIn',
```
Comment (prasanth-reddy-wavemaker-com) [inline website/siteConfig.js:77]: remove this from listing

**Comment by prasanth-reddy-wavemaker-com** [comment]
https://pr-184.dpwfbpxczhwpr.amplifyapp.com/learn/prefabs

**Comment by prasanth-reddy-wavemaker-com** [comment]
Wow!. Good start @swethakundaram 
The prefab specific page needs improvement. Each prefab page needs to have the following structutre.

1. Name
2. What functionality this prefab adds
3. How to configure it and use it
4. API published by prefab
5. Events published by prefab
6. Download the prefab

**Comment by swethakundaram** [comment]
Closing this pull request, please work on Prefabs documentation #216

**Pull request closed without merging**